satpy/composites/__init__.py

```diff
@@ -1068,6 +1074,11 @@ def __call__(self, projectables, **info):
         res.attrs['_FillValue'] = np.nan
         return res
 
+    @staticmethod
```

Comment:
staticmethod may make this a non-issue, but dask distributed usually has issues with map_blocks functions that it can't serialize (from top level of the module). I *think* this is fine, but wanted to mention it just in case.

Reply:
Good point, I'll make this a function just to be on the safe side.